packages/core/index.d.ts

```diff
@@ -8,6 +8,7 @@ declare type PluginHookWithMiddlewareName = (middlewareName: string) => void;
 declare type PluginHookPromise = (
 	request: Request,
 ) => Promise<unknown> | unknown;
+export type PluginExecutionMode = () => void;
```

Comment:
Do the explicit execution mode types need to be declared here as well? I got an import error when I tried to import them.

i.e.
```ts
export declare const executionModeDurableContext: PluginExecutionMode;
//...
```

Reply:
Thanks, I've added your suggestion for each of the execution modes.